PWGHF/Core/HfMlResponseDsToKKPi.h

```diff
@@ -68,6 +68,19 @@
     break;                                                                       \
   }
 
+// Variation of CHECK_AND_FILL_VEC_DS_HFHELPER(OBJECT, FEATURE, GETTER)
+// where GETTER1 and GETTER2 are methods of hfHelper, and the variable
+// is filled depending on whether it is a DsToKKPi or a DsToPiKK
+#define CHECK_AND_FILL_VEC_DS_OBJECT_HFHELPER_SIGNED(OBJECT1, OBJECT2, FEATURE, GETTER) \
```

Comment:
The `hfHelper` is not used in the macro, so maybe it could be removed from the title